src/main/webapp/js/index/form.js

```diff
@@ -18,34 +18,10 @@ $("#randomize-form").submit((event) => {
 });
 
 // Form underline element
-$("input, textarea").blur(() => {
+$("input, textarea").blur(function() {

```

Comment:
arrow functions are preferred

But you will need to address the $this - Ryan fixed it in his PR yesterday

Reply:
done